api/hearings/models/caseFlagReference.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface CaseFlagReferenceModel {
+  id: number;
+  flag_code: string;
+  value_en: string;
+  value_cy: string;
+  category_id: number;
+}
```

Comment:
It's not simple as this, we need to follow the same API contract from the Hearing Flags's api LLD.
Check the response Output payload in the:
https://tools.hmcts.net/confluence/display/CF/Case+Flags+HLD+Version+1.0
This needs re-work.